_packages/native-preview/bin/tsgo.js

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
+
+import getExePath from "#getExePath";
+import { execFileSync } from "node:child_process";
+
+const exe = getExePath();
+
+try {
+    execFileSync(exe, process.argv.slice(2), { stdio: "inherit" });
+}
+catch (e) {
+    if (e.status) {
+        process.exitCode = e.status;

```

Comment:
Should we immediately `process.exit` or is the convention to let potential callers "handle" something here?

Reply:
Yeah, it allows other stuff to finish up before exiting. Maybe it's pointless here.